docarray/array/storage/base/getsetdel.py

```diff
@@ -205,9 +205,8 @@ def _update_subindices_set(self, set_index, docs):
         else:  # root level set, update subindices iteratively
             for subindex_selector, subindex_da in subindices.items():
                 old_ids = DocumentArray(self[set_index])[subindex_selector, 'id']
-                with subindex_da:
-                    del subindex_da[old_ids]
-                    subindex_da.extend(DocumentArray(docs)[subindex_selector])
+                del subindex_da[old_ids]
+                subindex_da.extend(DocumentArray(docs)[subindex_selector])
```

Comment:
why this change @alaeddine-13 ?

Reply:
this is problematic for annlite. But in general I think subindices should be synced when the parent index is synced

Reply:
but why not use the context manager?